Clean/Utils/Bitwise.lean

```diff
@@ -1,7 +1,28 @@
 import Mathlib.Analysis.Normed.Ring.Lemmas
 import Clean.Utils.Field
+import Mathlib.Data.Nat.Bitwise
 
 namespace Bitwise
+
+section BitwiseAND
```

Comment:
namespace feels overkill to me here, these fit well with the other theorems in one namespace. but I'd prefer if you put theorems below definitions, i.e. move this further down below